go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/qdm12/ddns-updater
 
-go 1.17
+go 1.18
```

Comment:
Please keep it to 1.17 for now, since golangci-lint doesn't have all its linters updated to work with go 1.18.